index.js

```diff
@@ -1,14 +1,14 @@
 // @flow
 
 import "react-native-gesture-handler";
-
-import { AppRegistry } from "react-native";
-import inatjs from "inaturalistjs";
 import Config from "react-native-config";
+import inatjs from "inaturalistjs";
```

Comment:
This file reminded me, I think we should consider adding `eslint-plugin-simple-import-sort` to automatically sort imports from packages before imports from other files. That's one place I notice a lot of code inconsistency now that we have multiple devs working on this project, and I think sorting will make all files easier to read.

Reply:
Added this one too.